CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Fix bug with segment alarm whem processing a monthly mean collectiom
```

Comment:
Still not quite right.   There should be a section called `### Fixed` and this comment would go under that.   But I'll approve.  Matt will undoubtedly fix when there is a release.

Reply:
Yeah. Since I'm making a release as soon as it's in, it'll get fixed soon!

Reply:
I also spotted a typo: the world "collectiom" should have been collection. @mathomp4 could you fix it when making the release